.github/workflows/ci-coverage-build.yml

```diff
@@ -1,8 +1,9 @@
 name: Coverage Build
 on:
   workflow_dispatch:
-    branches:
-      - master
+    push:
+      branches:
+        - master
```

Comment:
Shouldn't the dispatch and push trigger be two separate entries?
```suggestion
    branches:
      - master
  push:
    branches:
      - master
```

Reply:
I think this is fine..

Reply:
we have it like tbis elsewhere too